### PR TITLE
docs: move from unpkg to jsdelivr for CDN install

### PR DESCRIPTION
**Pull request by luwes**

### CODE REVIEW

**Comment by luwes** [comment]
by the way the mux-audio and mux-video unpkg links weren't working 😅 

https://unpkg.com/@mux/mux-audio@0.4
https://unpkg.com/@mux/mux-video@0.4

File: packages/mux-audio/README.md [hunk 1]
```diff
@@ -44,13 +44,13 @@ require('@mux/mux-audio');
 Alternatively, use the CDN hosted version of this package:
 
 ```html
-<script src="https://unpkg.com/@mux/mux-audio@0.4"></script>
+<script src="https://cdn.jsdelivr.net/npm/@mux/mux-audio@0/dist/mux-audio.js"></script>
```
Comment (gkatsev) [inline packages/mux-audio/README.md:47]: if we wanted to, we can add a `jsdelivr` package field that points at this file to keep from needing to add the full path

**Comment by gkatsev** [comment]
> by the way the mux-audio and mux-video unpkg links weren't working 😅
> 
> https://unpkg.com/@mux/mux-audio@0.4 https://unpkg.com/@mux/mux-video@0.4

those versions don't exit on the `@mux` scope oops

File: packages/mux-audio/README.md (hunk 1 above)
Comment (luwes) [reply packages/mux-audio/README.md:47]: oh that would be neat, I'll add it to the PR later

Comment (gkatsev) [reply packages/mux-audio/README.md:47]: See https://www.jsdelivr.com/features#publishing-packages for more details

Comment (luwes) [reply packages/mux-audio/README.md:47]: 👍 added this

**Review by gkatsev — APPROVED** [review]

**Review by cjpillsbury — APPROVED** [review]

**Pull request merged**